TeapotTests/MockTests.swift

```diff
@@ -4,7 +4,8 @@ import XCTest
 class MockTests: XCTestCase {
 
     func testMock() {
-        let mockedTeapot = MockTeapot(bundle: Bundle(for: MockTests.self))
+        let mockedTeapot = MockTeapot(bundle: Bundle(for: MockTests.self), mockFileName: "get")
```

Comment:
maybe passing the full name `get.json` would make this API call clearer.

Reply:
But then i have to split the file name and the extension :( 

` if let url = currentBundle.url(forResource: resource, withExtension: "json") {`

So that would make it more error prone. I do agree it's more clear, but not sure its a good solution to do it like that...

Reply:
Or do you know a way to do this?

Reply:
Yeah, in that case, either get the extension from the API as well, replicating the `Bundle` API call, or leave it as is. 👍